packages/grant-explorer/src/features/discovery/hooks/useFilterRounds.ts

```diff
@@ -44,73 +56,97 @@ export enum RoundStatus {
 }
 
 export const ACTIVE_ROUNDS_FILTER: RoundSelectionParams = {
-  orderBy: "matchAmount",
-  orderDirection: "desc",
+  orderBy: "MATCH_AMOUNT_IN_USD_DESC",
   status: RoundStatus.active,
-  type: ROUND_PAYOUT_MERKLE_OLD,
+  type: ROUND_PAYOUT_MERKLE,
   network: "",
 };
 
 export const ROUNDS_ENDING_SOON_FILTER: RoundSelectionParams & {
   first: number;
 } = {
   first: 3,
-  orderBy: "roundEndTime",
-  orderDirection: "asc",
+  orderBy: "DONATIONS_END_TIME_ASC",
   type: "",
   network: "",
   status: RoundStatus.ending_soon,
 };
 
 export const useFilterRounds = (
-  filter: RoundSelectionParams,
+  where: RoundSelectionParams,
   chains: Chain[]
-): SWRResponse<__deprecated_RoundOverview[]> => {
+): SWRResponse<RoundGetRound[]> => {
+  const config = getConfig();
   const chainIds =
-    filter.network === undefined || filter.network.trim() === ""
+    where.network === undefined || where.network.trim() === ""
       ? chains.map((c) => c.id)
-      : filter.network.split(",").map(parseInt);
-  const statusFilter = createRoundsStatusFilter(filter.status);
+      : where.network.split(",").map((id) => parseInt(id));
+
+  const statusFilter = useMemo(
+    () => createRoundsStatusFilter(where.status),
+    [where.status]
+  );
   const strategyNames =
-    filter.type === undefined || filter.type.trim() === ""
+    where.type === undefined || where.type.trim() === ""
       ? []
-      : filter.type.split(",");
-  const where = createRoundWhereFilter(statusFilter, strategyNames);
+      : where.type.split(",");
+  const filter = createRoundWhereFilter(
+    statusFilter,
+    strategyNames,
+    chainIds,
+    config.allo.version
+  );
   const orderBy =
-    filter.orderBy === undefined || filter.orderBy === ""
-      ? "createdAt"
-      : filter.orderBy;
-  const orderDirection =
-    filter.orderDirection === undefined || filter.orderDirection === ""
-      ? "desc"
-      : filter.orderDirection;
-
-  return useRounds({ orderBy, orderDirection, where }, chainIds);
+    where.orderBy === undefined ? "CREATED_AT_BLOCK_DESC" : where.orderBy;
+  const vars = { orderBy, filter };
+  return useRounds(vars, chainIds);
 };
 
 const createRoundWhereFilter = (
-  statusFilter: __deprecated_TimestampVariables[],
-  strategyNames: string[]
-): __deprecated_RoundsQueryVariables["where"] => {
-  const payoutStrategy = strategyNames.length
-    ? strategyNames.map((strategyName) => ({ strategyName }))
-    : undefined;
-
+  statusFilter: TimeFilterVariables[],
+  strategyNames: string[],
+  chainIds: number[],
+  version: AlloVersion
+): RoundsQueryVariables["filter"] => {
   return {
+    // @ts-expect-error temp ignore
```

Comment:
is this still needed? it's doesn't explain why

Reply:
Yes, i added a better explanation